<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>537</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>537</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color:#292D3E;
}
QLabel{
font: bold 10pt &quot;Noto Sans KR&quot;;
color:white;
}
QLineEdit{
font: bold 10pt &quot;Noto Sans KR&quot;;
color:white;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page"/>
      <widget class="QWidget" name="page_2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QPushButton" name="BTN_OPEN">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
image:url(:/icon/icon/lidar-normal.jpg);
border: 0x;
}

QPushButton:hover{
image:url(:/icon/icon/lidar-over.jpg);
border: 0x;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BTN_CAPTURE">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
image:url(:/icon/icon/capture-normal.jpg);
border: 0x;
}

QPushButton:hover{
image:url(:/icon/icon/capture-over.jpg);
border: 0x;
}


QPushButton:pressed{
image:url(:/icon/icon/cpature-normal.jpg);
border: 0x;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BTN_SETTING">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
image:url(:/icon/icon/setting-normal.jpg);
border: 0x;
}

QPushButton:hover{
image:url(:/icon/icon/setting-over.jpg);
border: 0x;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BTN_CAPTURE1">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BTN_CAPTURE2">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BTN_THUMBNAIL">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
font: bold 12pt &quot;Noto Sans KR&quot;;
color:white;
border: 0px;
}

QPushButton:hover{
font: bold 12pt &quot;Noto Sans KR&quot;;
color:white;
background-color:rgba(255,255,255,30);
border: 0px;
}


QPushButton:pressed{
font: bold 12pt &quot;Noto Sans KR&quot;;
color:white;
background-color:rgba(255,255,255,100);
border: 0px;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QTabWidget" name="WORK_TAB">
            <property name="styleSheet">
             <string notr="true">
QTabBar::tab 
{
    background:rgb(59, 65, 89);
    color: #7F7F7F;   
	font: bold 11pt  &quot;Noto Sans KR&quot;;
    min-width: 184px;
    min-height: 20px;
    margin-left: 2px;
    margin-right: 1px;
	border-bottom: 5px solid rgba(255, 255, 255,20);
}

QTabWidget::pane {
border: 2px solid #393F4D;

}

QTabBar::tab:selected
{
	font: bold 11pt  &quot;Noto Sans KR&quot;;
	border-bottom: 5px solid #24ACAC;
    color: white;

}
QTabBar::tab:hover:!selected
{
    background:rgb(69, 75, 99);
    color: white;
	font: bold 11pt  &quot;Noto Sans KR&quot;;
	border-bottom: 5px solid rgba(255, 255, 255,20);
}

QTabBar::tab:pressed 
{
	font: 11pt  &quot;Noto Sans KR Black&quot;;
	border-bottom: 5px solid rgb(119, 199, 200);
    color: white;
    background:rgb(89, 95, 119);

}</string>
            </property>
            <property name="tabShape">
             <enum>QTabWidget::Rounded</enum>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <property name="documentMode">
             <bool>false</bool>
            </property>
            <widget class="QWidget" name="tab_4">
             <attribute name="title">
              <string>알고리즘 설정</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <property name="bottomMargin">
                 <number>15</number>
                </property>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_6">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="label_7">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">font: bold 14pt &quot;Noto Sans KR Medium&quot;;</string>
                    </property>
                    <property name="text">
                     <string>알고리즘 선택</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="Line" name="line_4">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>3</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>3</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">background-color:rgba(203, 244, 255, 50);</string>
                    </property>
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QLabel" name="LABEL_IMAGE_VIEW">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>576</width>
                    <height>324</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>576</width>
                    <height>324</height>
                   </size>
                  </property>
                  <property name="sizeIncrement">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border:1px solid gray</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="scaledContents">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <layout class="QGridLayout" name="gridLayout">
                  <item row="1" column="2">
                   <widget class="QSlider" name="SLIDER_EDGE_TH1">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>180</width>
                      <height>15</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QGroupBox{
font: bold 10pt &quot;Noto Sans KR&quot;;
}
QSlider::handle:horizontal {
    background:#24ACAC;
    height: 5px;
	width:10px;
	border-radius: 5px;
}

QSlider::handle:horizontal:hover {
    background:rgb(13, 226, 194);
    height: 5px;
	width:10px;
	border-radius: 5px;
}
QSlider::groove:horizontal {
	border-radius: 7px;
    background: rgba(50, 80, 50,150)
}

QSlider::handle:horizontal:disabled {
    background:rgb(38, 58, 65);
    height: 5px;
	width:10px;
	border-radius: 5px;
}
QSlider::groove:horizontal:disabled{
    background: rgba(60, 102, 121,30);
}
</string>
                    </property>
                    <property name="maximum">
                     <number>255</number>
                    </property>
                    <property name="value">
                     <number>150</number>
                    </property>
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0">
                   <widget class="QRadioButton" name="RADIO_ORIGINAL">
                    <property name="maximumSize">
                     <size>
                      <width>80</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">color:white;</string>
                    </property>
                    <property name="text">
                     <string>Original</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QRadioButton" name="RADIO_BINARY">
                    <property name="maximumSize">
                     <size>
                      <width>80</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">color:white;</string>
                    </property>
                    <property name="text">
                     <string>Binary</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="2">
                   <widget class="QRadioButton" name="RADIO_EDGE">
                    <property name="maximumSize">
                     <size>
                      <width>80</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">color:white;</string>
                    </property>
                    <property name="text">
                     <string>Edge</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QSlider" name="SLIDER_BIN_TH">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>180</width>
                      <height>15</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QGroupBox{
font: bold 10pt &quot;Noto Sans KR&quot;;
}
QSlider::handle:horizontal {
    background:#24ACAC;
    height: 5px;
	width:10px;
	border-radius: 5px;
}

QSlider::handle:horizontal:hover {
    background:rgb(13, 226, 194);
    height: 5px;
	width:10px;
	border-radius: 5px;
}
QSlider::groove:horizontal {
	border-radius: 7px;
    background: rgba(50, 80, 50,150)
}

QSlider::handle:horizontal:disabled {
    background:rgb(38, 58, 65);
    height: 5px;
	width:10px;
	border-radius: 5px;
}
QSlider::groove:horizontal:disabled{
    background: rgba(60, 102, 121,30);
}
</string>
                    </property>
                    <property name="maximum">
                     <number>255</number>
                    </property>
                    <property name="value">
                     <number>150</number>
                    </property>
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="2">
                   <widget class="QSlider" name="SLIDER_EDGE_TH2">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>180</width>
                      <height>15</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QGroupBox{
font: bold 10pt &quot;Noto Sans KR&quot;;
}
QSlider::handle:horizontal {
    background:#24ACAC;
    height: 5px;
	width:10px;
	border-radius: 5px;
}

QSlider::handle:horizontal:hover {
    background:rgb(13, 226, 194);
    height: 5px;
	width:10px;
	border-radius: 5px;
}
QSlider::groove:horizontal {
	border-radius: 7px;
    background: rgba(50, 80, 50,150)
}

QSlider::handle:horizontal:disabled {
    background:rgb(38, 58, 65);
    height: 5px;
	width:10px;
	border-radius: 5px;
}
QSlider::groove:horizontal:disabled{
    background: rgba(60, 102, 121,30);
}
</string>
                    </property>
                    <property name="maximum">
                     <number>255</number>
                    </property>
                    <property name="value">
                     <number>200</number>
                    </property>
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_5">
             <attribute name="title">
              <string>검출 수행</string>
             </attribute>
             <widget class="QLabel" name="LB_MSG">
              <property name="geometry">
               <rect>
                <x>17</x>
                <y>497</y>
                <width>211</width>
                <height>61</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">
background-color:rgba(0,0,0,50);
border-radius: 17px;
qproperty-alignment: AlignCenter;</string>
              </property>
              <property name="text">
               <string>Test message warning</string>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="tab_6">
             <attribute name="title">
              <string>파지 지점</string>
             </attribute>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="TE_LOG">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>500</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1500</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 2px solid #393F4D;
font: 10pt &quot;Noto Sans KR&quot;;
color: white</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
